src/D2L.Bmx/Program.cs

```diff
@@ -198,6 +201,15 @@
 // start bmx
 return await new CommandLineBuilder( rootCommand )
 	.UseDefaults()
+	.AddMiddleware( async (
+		context,
+		next
+	) => {
+		await UpdateChecker.CheckForUpdatesAsync( configProvider.GetConfiguration() );
+		await next( context );
+	},
+		System.CommandLine.Invocation.MiddlewareOrder.ExceptionHandler + 1
```

Comment:
I suppose this doesn't hurt, but this particular middle ware (the update checker) won't ever throw an exception so it seems kind of pointless to set this.

Reply:
Sorry this is about the placement in the middleware pipeline where this update checker is going to run. At the default order level, this wouldn't get run if a user were to run --version or --help

Or do you mean that the + 1 to have it run after the exception handling middleware is what could go?